### PR TITLE
R 24.05. Behaviour Twin MDP KIT (v1.0.0) revised

### PR DESCRIPTION
**Pull request by VincentMalik**
**Added**
extended KIT: Behaviour Twin MDP KIT, version 1.0.0.

- Adoption View finished
- Development View finished

- Added development view page including video. **Video is not working in local docusaurus!**
- Added asset discovery to development view.
- Added guideline for Implementing an FMU Interoperability Checker to development view.
- Added success stories section with MDP Application page.

### CODE REVIEW

File: docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+The objective is to enhance model-based design and data processing using models coming from component suppliers (for example: the OEM gets the models of the gear box, the brakes and the battery from the respective Tier1s), by facilitating model exchange between Tier1s and OEM or between Tier2s and Tier1. This is done using the industry standard FMU (which is a black box model, so the intellectual property is protected) and performing the exchange over EDC. The MDP KIT defines the functionality of the software applications needed for the MDP use cases, that is to combine the models into a simulation architecture and perform parametric studies, but also to do field data processing based on evaluation models of the component suppliers.
```
Comment (danielmiehle) [inline docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md:15]: I would suggest to link the FMU standard.

File: docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## General Information Display: BehBehaviouravior Twin MDP Application
+
+The General Information Display of the Behaviour Twin MDP (Model and Data Processing) Application offers users a user-friendly interface to visualize essential information about behavioural digital twins and their components. Whether users are analyzing the behaviour of a component within a vehicle or maintaining a system, this feature provides easy access to valuable insights.
+
+A technical expert from the OEM has the task to build a design of a new car. The simulation expert will collect the necessary models from the Tier1 and combine them in a vehicle model to evaluate different designs. The models are exchanged between Catena-X partners using EDC and using the FMU format (black box model).
+
+![Overview Application](../images/Pic_AppStory1.png)
```
Comment (danielmiehle) [inline docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md:23]: 1. Do you have the confirmation that you are allowed to use the Siemens Logo?
2. In addition the app screenshots look like powerpoint - is this a real application?
3. How is the MDP app differentiated from the Siemens apps? Can you please explain this in the introduction.

File: docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+The objective is to enhance model-based design and data processing using models coming from component suppliers (for example: the OEM gets the models of the gear box, the brakes and the battery from the respective Tier1s), by facilitating model exchange between Tier1s and OEM or between Tier2s and Tier1. This is done using the industry standard FMU (which is a black box model, so the intellectual property is protected) and performing the exchange over EDC. The MDP KIT defines the functionality of the software applications needed for the MDP use cases, that is to combine the models into a simulation architecture and perform parametric studies, but also to do field data processing based on evaluation models of the component suppliers.
+
+## General Information Display: BehBehaviouravior Twin MDP Application
```
Comment (danielmiehle) [inline docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md:17]: Typo: "BehBehaviouravior"

File: docs-kits/kits/Behaviour Twin MDP Kit/images/mdp_kit_v1.0.mp4
Comment (danielmiehle) [inline docs-kits/kits/Behaviour Twin MDP Kit/images/mdp_kit_v1.0.mp4:1]: Non-blocking: Videos should be uploaded to the Catena-X YouTube Chanel and then linked / integrated. 

@HFocken @jSchuetz88 @stephanbcbauer: can you please advise? 
Or just contact info@catena-x.net

File: docs-kits/kits/Behaviour Twin MDP Kit/page_development-view.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+description: Model Based Development and Data Processing Kit
+---
+
+![Model Based Development and Data Processing Kit banner](/img/kit-icons/behaviour-mdp-kit-icon.svg)
+
+### Model Based Development and Data Processing Kit
+
+## MDP Development Video
+
+This video demonstrates the necessary software application functionality for implementing the MDP Use Cases:
```
Comment (danielmiehle) [inline docs-kits/kits/Behaviour Twin MDP Kit/page_development-view.md:13]: The development view should remain vendor-neutral. Therefore, please (1) move the video and (2) all Siemens related description to the success story or re-write the section in a vendor-neutral way. Thanks!

**Review by danielmiehle — CHANGES_REQUESTED** [review]
Thank you all for the valuable KIT addition. Please make sure that the KITs remain vendor-neutral - see comments.

**Comment by danielmiehle** [comment]
@maximilianong @HFocken @jSchuetz88 : I created an issues to add guidelines for the integration of videos in KITs.

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/issues/892

File: docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md (hunk 1 above)
Comment (AndreasSchwerin) [reply docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md]: Link has been added to the Catena-X standard CX-0102

File: docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md (hunk 2 above)
Comment (AndreasSchwerin) [reply docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md:23]: 1. Yes, this has been clarified.
2. This is a real App, based on the Mendix-Tool Suite. Of course, it contains on some of the home screens images taken from Powerpoint (the smaller rectangles on the pages).
3. The MDP app is a proof-of-concept app developed within the Catena-X project to illustrate the MDP work flow and a possible implementation of a user guidance. This is not a commercial Siemens product. This is mentioned now in the introduction section of the MDP app success story.

File: docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md (hunk 3 above)
Comment (AndreasSchwerin) [reply docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md]: has been corrected, thank you.

File: docs-kits/kits/Behaviour Twin MDP Kit/images/mdp_kit_v1.0.mp4
Comment (AndreasSchwerin) [reply docs-kits/kits/Behaviour Twin MDP Kit/images/mdp_kit_v1.0.mp4:1]: Thanks for the hint. We asked at info@catena-x.net to upload the video to the Catena-X Youtube Channel (after they offered this to do so). The link to the video will be included as soon as we have it.

File: docs-kits/kits/Behaviour Twin MDP Kit/page_development-view.md (hunk 4 above)
Comment (AndreasSchwerin) [reply docs-kits/kits/Behaviour Twin MDP Kit/page_development-view.md]: OK, we removed the video and other Siemens related content from the Development View and moved it to the Success Stories section.

**Comment by AndreasSchwerin** [comment]
We have uploaded an updated [version](https://github.com/catenax-ng/eclipse-tractusx.github.io/commit/7d19ba15066c65d0affaf4da4c57d6d6ad707caa), which in our view is complete, except for the video link.

**Comment by AndreasSchwerin** [comment]
@danielmiehle eclipsefdn/eca failed for olexiy-kupriyanov, because first time he had no signed ECA. However after signature of the ECA, we still get the same error message.
What should we do now?

**Review by HFocken — APPROVED** [review]
Approved under the condition that Daniels Feedback is considered! Thank you for your work!!

**Review by jSchuetz88 — APPROVED** [review]
I agree to the comments already made, please address them. 
But since my own comments have been included, an approval from my side.

File: docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md (hunk 2 above)
Comment (danielmiehle) [reply docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md:23]: Thanks for the clarification!

**Comment by danielmiehle** [comment]
> @danielmiehle eclipsefdn/eca failed for olexiy-kupriyanov, because first time he had no signed ECA. However after signature of the ECA, we still get the same error message. What should we do now?

@AngelikaWittek : Can you please help?

**Review by danielmiehle — APPROVED** [review]

**Comment by AndreasSchwerin** [comment]
Thanks for the approvals so far.
We still wait for the video upload to include the link in the success story.

**Comment by danielmiehle** [comment]
Thanks for your updates. Ready for merge!

**Comment by AndreasSchwerin** [comment]
@maximilianong @danielmiehle, We asked at info@catena-x.net to upload the success story video to the Catena-X Youtube Channel, we also contacted Lamei Tang directly, but up to now we did not receive the missing link or an answer. What can we do?

@maximilianong, we also still would need your approval.

**Review by maximilianong — APPROVED** [review]

**Comment by olexiy-kupriyanov** [comment]
@maximilianong, @danielmiehle, thanks a lot for the approvals and the merge. We just received the link to the Youtube channel, what would be the best way to update it now? Do we need a new pull request for that?

**Pull request merged**